events/mbed_events.h

```diff
@@ -30,6 +30,60 @@
 
 using namespace events;
 
+namespace mbed {
+
+/**
+ * Return a pointer to an EventQueue, on which normal tasks can be queued.
+ *
+ * All calls to this return the same EventQueue - it and its dispatch thread
+ * are created on the first call to this function. The dispatch thread
+ * runs at default priority (currently osPriorityNormal).
+ *
+ * The EventQueue returned may be used to call() Events, or to chain() other
+ * EventQueues so that they are run in the same context.
+ *
+ * Events (or chained EventQueues) executing on the normal event queue should
+ * normally take less than 10ms to execute, to avoid starving other users. As
+ * such, users can expect that event latency will typically be 10ms or less,
+ * but could occasionally be significantly higher if many events are queued.
+ *
+ * If an RTOS is not present or the configuration option
+ * `events.shared-normal-dispatch-from-application` is set to true, then this
+ * does not create a dedicated dispatch thread - instead the application is
+ * expected to run the EventQueue's dispatch, eg from main. This is necessary
+ * for the event loop to work without an RTOS, or an RTOS system can can save
+ * memory by reusing the main stack.
+ *
+ * @return pointer to normal event queue
+ */
+events::EventQueue *get_shared_normal_event_queue();
+
+#ifdef MBED_CONF_RTOS_PRESENT
+/**
+ * Return a pointer to an EventQueue, on which small tasks to be deferred from
+ * interrupt can be queued.
+ *
+ * All calls to this return the same EventQueue - it and its thread are
+ * created on the first call to this function. The dispatch thread
+ * runs at a high priority (currently osPriorityRealtime).
+ *
+ * The EventQueue returned may be used to call() Events, or to chain() other
+ * EventQueues so that they are run in the same context.
+ *
+ * Events (or chained EventQueues) executing on the normal event queue should
+ * normally take less than 100us to execute, to avoid starving other users. As
+ * such, users can expect that event latency will typically be 100us or less,
+ * but could occasionally be significantly higher if many events are queued.
+ *
+ * @return pointer to interrupt deferral event queue
+ */
+
+events::EventQueue *get_shared_interrupt_deferral_event_queue();
```

Comment:
BTW, did you have a view on my previous thought that it could be named from the latency point of view - "high-priority / low-latency / realtime" or something?